type: Pr2GripperController
joint_name: r_gripper_joint
fingertip_sensor_topic: /pressure/r_gripper_motor